mobile/app/lib/byzcoin-def.ts

```diff
@@ -14,13 +13,14 @@ import { StatusRequest, StatusResponse } from "@dedis/cothority/status/proto";
 import StatusRPC from "@dedis/cothority/status/status-rpc";
 import { Point } from "@dedis/kyber";
 import { bdDEDIS, bdTest } from "~/lib/byzcoin-defs";
+import { Data, PersonhoodRPC } from "@c4dt/dynacred";
 import { StorageFile } from "~/lib/storage-file";
 import { TestData } from "~/lib/test-data";
 import { setNodeList } from "~/pages/settings/settings-page";
 
 // Which byzcoinDefinition to use.
-export const bcDef = bdDEDIS;
-// export const bcDef = bdTest;
+// export const bcDef = bdDEDIS;
```

Comment:
I'll need to make sure that travis checks this - but by default, `bdDEDIS` should be activated, so that we can create a mobile package directly.